frontend/src/pages/Contracts/Contract/Resources/index.tsx

```diff
@@ -21,11 +21,16 @@ export const ContractResources = () => {
     deleteOne: `contracts/resources/{id}`,
   };
 
+  const columnWidths = {
+    resource: 2,
+  };
+
   return (
     <TableComplete
-      itemName={"resource"}
+      itemName={"Resources"}
```

Comment:
I would use another term for the variable instead of capitalizing it.

Reply:
just wondering what your concern is here?